### ISSUE TITLE
Create tor hidden service in install-globaleaks.sh

### ISSUE THREAD

**Issue by combs** [globaleaks/GlobaLeaks#1884]
~                           

**Current behavior**

No hidden service is created during install, though the Globaleaks Install Guide refers to one.

**Expected behavior**

Lines like these are added to `torrc`
```
HiddenServiceDir /var/lib/tor/globaleaks/
HiddenServicePort 80 127.0.0.1:80
```
Restart tor and output the hostname as part of `install-globaleaks.sh`

**Minimal reproduction of the problem with instructions**

run `install-globaleaks.sh` on vanilla machine

**What is the motivation / use case for changing the behavior?**

* **GlobaLeaks version:** 2.X.Y

GlobaLeaks Version: 2.65.14
GlobaLeaks DB version: 35

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Other (specify) ]

n/a

**Comment by combs** [comment]
It looks like maybe there are some hooks already?
```
if [ -r /var/globaleaks/torhs/hostname ]; then
  TORHS=`cat /var/globaleaks/torhs/hostname`
  echo "To access your GlobaLeaks use the following Tor HS URL: $TORHS"
  echo "Use Tor Browser to access it, download it from https://antani.tor2web.org/gettor"
  echo "If you need to access it directly on your public IP address, you must edit /etc/default/globaleaks and restart globaleaks"
fi

```

`/var/globaleaks/torhs/` gets made but was never populated on my machine.

**Comment by evilaliv3** [comment]
Hello there @combs.

As reported on https://github.com/globaleaks/GlobaLeaks/issues/1883 could you just trying to disable Apparmor?

Could you please also add the output of dmesg here. I'm quite sure that you will have some issues on apparmor related to this as well.

**Comment by evilaliv3** [comment]
Yep i conirm that this is quite for sure related to the apparmor misconfiugration for the operating system adopted.

@combs could you please remember me your exact version of the operating system so to annotate it here on the ticket and change the name of the ticket accordingly? thanks!
